docs/examples/standalone_rpc.py

```diff
@@ -0,0 +1,8 @@
+from nameko.standalone.rpc import ClusterRpcProxy
+
+config = {
+    'AMQP_URI': 'amqp://guest:guest@localhost/nameko'
+}
+
+with ClusterRpcProxy(config) as cluster_rpc:
+    cluster_rpc.service_x.remote_method("hello")  # "x-y-hello"
```

Comment:
don't love this as an example

also, for this, where the caller example is tightly coupled to the service example, it might be better to keep them in the same file (e.g. here i think the response output has drifted out of sync with the method)